src/Domain/Entities/Interviews/InterviewShareLink.cs

```diff
@@ -0,0 +1,29 @@
+﻿using System;
+
+namespace Domain.Entities.Interviews
+{
+    public class InterviewShareLink : IHasId
+    {
+        public InterviewShareLink()
```

Comment:
```suggestion
        protected InterviewShareLink()
```

ЧТоб никто не смог создать сущность с каким угодно набором полей. DDD принцип